runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/layouts/Layout.java

```diff
@@ -17,12 +17,6 @@
 
 public abstract class Layout implements AutoCloseable
 {
-    public enum Mode
-    {
-        CREATE_READ_WRITE,
-        READ_ONLY
-    }
-
```

Comment:
Can this class be removed now?

Reply:
I am not sure about removing this class. It's the base class `BudgetsLayout`, `StreamsLayout`, `BufferPoolLayout` as well as `ScalarsLayout` and `HistogramsLayout`. Pls LMK.

Reply:
I think we can safely remove this as it no longer serves much purpose.

Each implementation would now directly implement `AutoCloseable` interface and `Builder` inner class can be removed as superclass from `Builder` implementations.

Reply:
Yep, got it. Thx for clarifying. Done, pls chk.